WalletWasabi/Nostr/NostrExtensions.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System.Linq;
+using System.Net;
+using System.Net.WebSockets;
+using System.Threading;
+using System.Threading.Tasks;
+using NBitcoin;
+using NBitcoin.Secp256k1;
+using NNostr.Client;
+using WalletWasabi.Exceptions;
+using WalletWasabi.Helpers;
+
+namespace WalletWasabi.Nostr;
+
+public record NostrCoordinator(string Description, string Name, Uri Uri, Network Network)
+{
+	private static NostrCoordinator Ginger = new(
+		Description: "Ginger Coordinator | FREE Remix, FREE under 0.01 BTC, FREE for Wasabi mixed coins | Secure Coinjoin - Illicit actors are not allowed to participate!",
```

Comment:
Review the description please

Reply:
how about: official coordinator, visit gingerwallet.io!

Reply:
This is how others advertise themself atm:

![image](https://github.com/GingerPrivacy/GingerWallet/assets/16364053/b3d16578-3508-4797-8acc-c02f7d7049d1)

Reply:
Pushed